### DISCUSSION TITLE
[Bug] The remote server returned an error: (401) Unauthorized.

### DISCUSSION THREAD

**Discussion by keyrad-abasi** [ScoopInstaller/Scoop#5879] [Help and Questions]
## Bug Report
Any attempt to update the following list of packages results in error:
- gh
- git
- telegram
- windows-terminal

#### Current Behavior
Errors with this message: **The remote server returned an error: (401) Unauthorized.**

#### Expected Behavior
It should be updated without any error.

#### Additional context/output
Running both in windows-terminal and command prompt results in following error:

```windows-terminal
C:\Users\keyrada>scoop update git
git: 2.42.0.2 -> 2.43.0
Updating one outdated app:
Updating 'git' (2.42.0.2 -> 2.43.0)
Downloading new version
The remote server returned an error: (401) Unauthorized.
At C:\Users\keyrada\scoop\apps\scoop\current\lib\install.ps1:111 char:9
+         throw $e
+         ~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], WebException
    + FullyQualifiedErrorId : The remote server returned an error: (401) Unauthorized.
```
```windows-terminal
C:\Users\keyrada>scoop update gh
gh: 2.38.0 -> 2.39.2
Updating one outdated app:
Updating 'gh' (2.38.0 -> 2.39.2)
Downloading new version
The remote server returned an error: (401) Unauthorized.
At C:\Users\keyrada\scoop\apps\scoop\current\lib\install.ps1:111 char:9
+         throw $e
+         ~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], WebException
    + FullyQualifiedErrorId : The remote server returned an error: (401) Unauthorized.
```
### System details

**Windows version:** 10 Enterprise, OS build: 19045.3570

**OS architecture:** 64bit

**PowerShell version:**  
| Major       | Minor       | Build  | Revision |
|------------|------------|--------|----------|
| 5              | 1               | 19041 | 3570      |

**Additional software:** Git

#### Scoop Configuration
```json
{
    "last_update":  "2023-12-01T04:37:49.0323743+03:30",
    "gh_token":  "github-pat-XXX",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}
```

**Comment by HUMORCE** [comment]
Are you meet this issue with VPN/corporate network?

**Comment by 1036831050** [comment]
我也遇到了这个问题，win11，突然出现的。
报错截图
![image](https://github.com/ScoopInstaller/Scoop/assets/38917394/1a926e49-0d9e-4ead-a0e8-d9a1914cca31)
config
![image](https://github.com/ScoopInstaller/Scoop/assets/38917394/e3a69eac-71d7-439b-a2b9-97628deba528)

**Comment by 1036831050** [comment]
> ## Bug Report
> Any attempt to update the following list of packages results in error:
> 
> * gh
> * git
> * telegram
> * windows-terminal
> 
> #### Current Behavior
> Errors with this message: **The remote server returned an error: (401) Unauthorized.**
> 
> #### Expected Behavior
> It should be updated without any error.
> 
> #### Additional context/output
> Running both in windows-terminal and command prompt results in following error:
> 
> ```
> C:\Users\keyrada>scoop update git
> git: 2.42.0.2 -> 2.43.0
> Updating one outdated app:
> Updating 'git' (2.42.0.2 -> 2.43.0)
> Downloading new version
> The remote server returned an error: (401) Unauthorized.
> At C:\Users\keyrada\scoop\apps\scoop\current\lib\install.ps1:111 char:9
> +         throw $e
> +         ~~~~~~~~
>     + CategoryInfo          : OperationStopped: (:) [], WebException
>     + FullyQualifiedErrorId : The remote server returned an error: (401) Unauthorized.
> ```
> 
> ```
> C:\Users\keyrada>scoop update gh
> gh: 2.38.0 -> 2.39.2
> Updating one outdated app:
> Updating 'gh' (2.38.0 -> 2.39.2)
> Downloading new version
> The remote server returned an error: (401) Unauthorized.
> At C:\Users\keyrada\scoop\apps\scoop\current\lib\install.ps1:111 char:9
> +         throw $e
> +         ~~~~~~~~
>     + CategoryInfo          : OperationStopped: (:) [], WebException
>     + FullyQualifiedErrorId : The remote server returned an error: (401) Unauthorized.
> ```
> 
> ### System details
> **Windows version:** 10 Enterprise, OS build: 19045.3570
> 
> **OS architecture:** 64bit
> 
> **PowerShell version:**
> 
> Major	Minor	Build	Revision
> 5	1	19041	3570
> **Additional software:** Git
> 
> #### Scoop Configuration
> ```json
> {
>     "last_update":  "2023-12-01T04:37:49.0323743+03:30",
>     "gh_token":  "github-pat-XXX",
>     "scoop_branch":  "master",
>     "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
> }
> ```

我重新设置了一下gh_token恢复了，你可以试下
